scripts/get-resource-index-ref-matrix

```diff
@@ -100,7 +109,15 @@ get_resource_index_at_commit() {
     # script. We can remove it once all the servers have been updated.
     #   -Jover, 01 May 2024
     if [[ -f "$repo/scripts/get-resource-index.js" ]]; then
-        npm ci --silent --prefix "$repo"
+        >&2 echo "Running $repo/scripts/get-resource-index.js"
+        >&2 echo "RESOURCE_INDEX in $repo/env/production/config.json: $(jq -r '.RESOURCE_INDEX' $repo/env/production/config.json)"
+
+        if [[ -n "${RESOURCE_INDEX}" ]]; then
+            >&2 echo "RESOURCE_INDEX envvar: $RESOURCE_INDEX"
+        fi
+
+        cd "$repo"
+        npm ci --silent
```

Comment:
Why do we need to run `npm ci` at all? All that script needs is `node`...

Did you figure out _why_ the old code (`npm ci --silent --prefix "$repo"`) resulted in the bug?